templates/rhsm-katello-reconfigure.erb

```diff
@@ -63,6 +63,17 @@ else
   sed -i "s/baseurl.*/&\n\n$full_refresh_config/g" $CFG
 fi
 
+# also add the katello ca cert to the system wide ca cert store
+if [-d $CA_TRUST_ANCHORS]; then
+  update-ca-trust enable
+  cp %%(ca_cert_dir)skatello-server-ca.pem $CA_TRUST_ANCHORS
```

Comment:
The name of the cert needs to be set based on the parameter lest it not work when someone configures it -- https://github.com/Katello/puppet-certs/blob/master/manifests/init.pp#L111